source/docs/software/actuators/overview.rst

```diff
@@ -1,32 +1,19 @@
 Actuator Overview
 ==================
-This section discusses the control of motors and pneumatics
-through speed controllers, solenoids and pneumatics, and
-their interface with C++ and Java WPILib.
+This section discusses the control of motors and pneumatics through speed controllers, solenoids and pneumatics, and their interface with C++ and Java WPILib.
 
 Speed controllers
 -----------------
-A speed controller is responsible on your robot for making motors move.
-For brushed DC motors such as CIMs or 775s, the speed controller regulates
-the voltage that the motor receives, much like a light bulb. For
-brushless speed controllers such as the Spark MAX, the controller regulates
-the power delivered to each "phase" of the motor.
+A speed controller is responsible on your robot for making motors move. For brushed DC motors such as CIMs or 775s, the speed controller regulates the voltage that the motor receives, much like a light bulb. For brushless speed controllers such as the Spark MAX, the controller regulates the power delivered to each "phase" of the motor.
 
 .. .. hint::
-..     One can make a quick, non-competition-legal speed controller by
-..     removing the motor from a cordless BRUSHED drill and attaching
-..     PowerPoles or equivalents to the motor's leads. Make sure that
-..     the voltage supplied by the drill will not damage the motor,
-..     but note that the 775 is fine at up to 24 volts.
+..     One can make a quick, non-competition-legal speed controller by removing the motor from a cordless BRUSHED drill and attaching PowerPoles or equivalents to the motor's leads. Make sure that the voltage supplied by the drill will not damage the motor, but note that the 775 is fine at up to 24 volts.
 
-.. warning::
-    Connecting a BRUSHLESS motor controller straight to power, such as
-    to a conventional brushed motor controller, will destroy the motor!
+.. warning:: Connecting a BRUSHLESS motor controller straight to power, such as to a conventional brushed motor controller, will destroy the motor!
 
 FRC Legal Motor Controllers
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Speed controllers come in lots of shapes, sizes and feature sets. This
-is the full list of FRC Legal speed controllers as of January 2020:
+Speed controllers come in lots of shapes, sizes and feature sets. This is the full list of FRC Legal speed controllers as of January 2020:
```

Comment:
Could [this link](https://www.firstinspires.org/robotics/frc/blog/2020-motors-controllers-some-robot-rules) be added?

```suggestion
Speed controllers come in lots of shapes, sizes and feature sets. The full list of FRC Legal speed controllers is detailed in this `official blog post <https://www.firstinspires.org/robotics/frc/blog/2020-motors-controllers-some-robot-rules>`__ and below, as of April 2020:
```

Reply:
Some years there is a blog post and some years there is not a blog post.

Reply:
I guess we could leave the most recent blog post linked? I'm not sure it would be helpful, just asking...

Reply:
I do not think it is necessary.